NetworkConfigurations/NetworkAddressingAndFilteringSubCategories/PortTranslation.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Port Translation
+grand_parent: Network Configuration
+parent: Network Addressing and Filtering
+nav_order: 6
+layout: default
+---
+
+# Port Translation
```

Comment:
Please add a graph of sample topology (you can use VMs) and show the config works (i.e. add a simple command like ping and show it is successful.)